composer.json

```diff
@@ -159,9 +159,9 @@
 		"ci:tests:create-directories": "mkdir -p .Build/public/typo3temp/var/tests",
 		"ci:tests:functional": [
 			"@ci:tests:create-directories",
-			"find 'Tests/Functional' -wholename '*Test.php' | parallel --gnu 'echo; echo \\\"Running functional test suite {}\\\"; .Build/bin/phpunit -c Build/phpunit/FunctionalTests.xml {}';"
+			"find 'Tests/Functional' -wholename '*Test.php' | parallel --gnu 'echo; echo \\\"Running functional test suite {}\\\"; phpunit -c Build/phpunit/FunctionalTests.xml {}';"
```

Comment:
For the cases where we are piping something to a Composer script (in this case: functional tests), we'll need to keep using the full path AFAIK.

Reply:
Might be can not really test this because of parallel error